Content.Server/GameObjects/Components/MachineLinking/IReceiver.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Robust.Shared.GameObjects;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Content.Server.GameObjects.Components.MachineLinking
+{
+    public interface IReceiver
```

Comment:
I wonder if we could make this an `IReceiver<T>` so triggering could take any argument, hmm.